package.json

```diff
@@ -34,81 +34,85 @@
   },
   "homepage": "https://github.com/bigcommerce/stencil-cli",
   "dependencies": {
-    "@bigcommerce/stencil-paper": "^2.0.2",
+    "@bigcommerce/stencil-paper": "^2.0.4",
     "@bigcommerce/stencil-styles": "1.1.0",
-    "accept-language-parser": "^1.0.2",
-    "archiver": "^0.14.4",
+    "accept-language-parser": "^1.4.1",
+    "archiver": "^2.0.0",
     "async": "^2.4.0",
     "babel-eslint": "^7.1.1",
-    "boom": "^2.7.0",
+    "boom": "^5.2.0",
     "browser-sync": "git://github.com/mcampa/browser-sync.git#tunnelBug",
-    "cheerio": "^0.19.0",
+    "cheerio": "^1.0.0-rc.2",
     "code": "^4.0.0",
     "colors": "^1.0.3",
     "commander": "^2.7.1",
-    "confidence": "^1.0.0",
+    "confidence": "^3.0.2",
     "dateformat": "^2.0.0",
-    "decompress-zip": "^0.2.0",
-    "eslint": "^2.2.0",
-    "eslint-config-airbnb": "^6.0.2",
-    "front-matter": "^1.0.0",
-    "github": "^8.1.0",
-    "glob": "^5.0.14",
-    "glue": "^2.0.0",
-    "good": "^5.1.2",
-    "good-console": "^4.1.0",
-    "handlebars": "^3.0.3",
-    "hapi": "^8.4.0",
-    "hoek": "^2.12.0",
-    "image-size": "^0.4.0",
-    "inquirer": "^0.8.2",
-    "install": "^0.1.8",
+    "decompress-zip": "^0.3.0",
+    "eslint": "^4.5.0",
+    "eslint-config-airbnb": "^15.1.0",
+    "eslint-plugin-react": "^7.3.0",
+    "front-matter": "^2.1.2",
+    "github": "^9.3.1",
+    "glob": "^7.1.2",
+    "glue": "^4.2.0",
+    "good": "^7.2.0",
+    "good-console": "^6.4.0",
+    "h2o2": "^6.0.0",
+    "handlebars": "^4.0.10",
+    "hapi": "^16.5.2",
```

Comment:
:)  yeah, it wasn't so bad, just had to change how the plugins are loaded and some of the handlers (like 'directory' for example) are now in Hapi plugins